src/metatensor/models/cli/eval_model.py

```diff
@@ -1,11 +1,31 @@
 import argparse
+import logging
+from typing import Dict, Tuple, Union
 
-from ..utils.data.readers import read_structures
-from ..utils.data.writers import write_predictions
+import torch
+from omegaconf import OmegaConf
+
+from ..utils.compute_loss import compute_model_loss
+from ..utils.data import (
+    Dataset,
+    collate_fn,
+    read_structures,
+    read_targets,
+    write_predictions,
+)
+from ..utils.extract_targets import get_outputs_dict
+from ..utils.info import finalize_aggregated_info, update_aggregated_info
+from ..utils.loss import TensorMapDictLoss
 from ..utils.model_io import load_model
+from ..utils.omegaconf import _has_yaml_suffix, expand_dataset_config
 from .formatter import CustomHelpFormatter
 
 
+logger = logging.getLogger(__name__)
+
+CHOICES_EVAL_ON = ["training", "validation", "test"]
```

Comment:
Not a big fan. Here IMO the datasets should have custom names given by the user. I might want to evaluate the "universal" PET model on my toy "silicon" dataset and on a "water" dataset, which will be read from files, but training, validation, test don't make much sense

Reply:
I see your point. But then people have to write a new `.yaml` file? Is this what we want?